### ISSUE TITLE
gulp.src with multiple directories

### ISSUE THREAD

**Issue by idiotWu** [smysnk/gulp-rev-all#122]
I want to process revision on files in multiple directories:

```javascript
gulp.src(['build/scripts/**/*', 'src/assets/**/*'])
    .pipe(revAll.revision())
    .pipe(gulp.dest('dist/'));
```

The output is expected to be:

```
dist
|_ scripts
   |_ somejs.3c4031a2.js
|_  assets
   |_ someimg.k4k019a1.png
```

However the output now is:

```
dist
|_ build
   |_ scripts
       |_ somejs.3c4031a2.js
|_ src
   |_ assets
       |_ someimg.k4k019a1.png
```

**Comment by aeomer** [comment]
The current behaviour looks correct to me. It would be difficult to see how the engine would know you want to strip the first part of your source target and replace it with your preferred initial path. Indeed, it would quite complicated to present that in options.
tl;dr
However, I needed something similar and so I used what I call my 'parade-ground' where all the assets are copied and then revisioned.  This gives me freedom to pull assets from many locations, including templates and files created in other projects by UI/UX people, and put them together before generating revisions, uglyfying them etc. The solution requires some forced sequencing and is beyond the scope of a simple answer, but assembling everything into one place before versioning is my solution. I have watchers looking for changes in the external projects which bring them into the 'parade-group' directory and then run the revision task.

**Comment by idiotWu** [comment]
Thanks for the advise, that's exactly what I am doing now :)

**Comment by aeomer** [comment]
You're welcome.
